src/components/AppNavigation/Settings/ShortcutOverview.vue

```diff
@@ -8,17 +8,17 @@
 			v-for="category in shortcuts"
 			:key="category.categoryId"
 			class="shortcut-section">
-			<h3 class="shortcut-section__header">
+			<h6 class="shortcut-section__header">
 				{{ category.categoryLabel }}
-			</h3>
+			</h6>
 			<div
 				v-for="(shortcut, index) in category.shortcuts"
 				:key="`${category.categoryId}-${index}`"
 				class="shortcut-section-item">
 				<span class="shortcut-section-item__keys">
 					<template v-for="(keyCombination, index2) of shortcut.keys">
 						<template v-for="(key, index3) in keyCombination">
-							<kbd :key="`${category.categoryId}-${index}-${index2}-${index3}`">{{ key }}</kbd>
+							<NcKbd :key="`${category.categoryId}-${index}-${index2}-${index3}`">{{ key }}</NcKbd>
```

Comment:
In addition to using `NcAppSettingsShortcurtsSection`, `<NcKbd>` has the slot only for special cases (advanced usage). But the preferred way is to use the `symbol` prop.

It has a lot of special handling that otherwise must be handled by the app:
- Translating key names like `Ctrl` to `Strg` in German
- Using macOS names like `Option` instead of `Alt`
- Using `Command` instead of `Ctrl` on macOS
- Showing symbols from `↑` for ArrowUp to `⌥` for Alt (Option) on macOS

https://nextcloud-vue-components.netlify.app/#/Components/NcKbd